infra/main.bicep

```diff
@@ -276,3 +276,75 @@ resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2022-10
   name: 'log-${suffix}'
   location: location
 }
+
+resource deplymentAddDataSource 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  name: 'deployment-add-data-source'
+  location: location
+  kind: 'AzureCLI'
+  properties: {
+    azCliVersion: '2.42.0'
+    timeout: 'PT5M'
+    forceUpdateTag: '5'
+    containerSettings: {
+      containerGroupName: 'deployment-add-data-source'
+    }
+    retentionInterval: 'PT1H'
+    cleanupPreference: 'OnExpiration'
+    environmentVariables: [
+      {
+        name: 'CONNECTION_STRING'
+        secureValue: 'jdbc:postgresql://${atlasDatabase.outputs.postgresServerFullyQualifiedDomainName}:5432/${postgresOMOPCDMDatabaseName}?user=postgres_admin&password=${postgresOMOPCDMpassword}&sslmode=require'
+      }
+      {
+        name: 'OHDSI_WEBAPI_PASSWORD'
+        secureValue: atlasSecurityAdminPassword
+      }
+      {
+        name: 'OHDSI_WEBAPI_USER'
+        value: 'admin'
+      }
+      {
+        name: 'OHDSI_WEBAPI_URL'
+        value: ohdsiWebApiWebapp.outputs.ohdsiWebapiUrl
+      }
+      {
+        name: 'DIALECT'
+        value: 'postgresql'
+      }
+      {
+        name: 'SOURCE_NAME'
+        value: 'omop-cdm-synthea'
+      }
+      {
+        name: 'SOURCE_KEY'
+        value: 'omop-cdm-synthea'
+      }
+      {
+        name: 'USERNAME'
+        value: atlasDatabase.outputs.postgresWebapiAdminUsername
+      }
+      {
+        name: 'PASSWORD'
+        secureValue: postgresWebapiAdminPassword
+      }
+      {
+        name: 'DAIMON_CDM'
+        value: 'cdm'
+      }
+      {
+        name: 'DAIMON_VOCABULARY'
+        value: 'cdm'
+      }
+      {
+        name: 'DAIMON_RESULTS'
+        value: 'cdm_results'
+      }
+    ]
+    scriptContent: loadTextContent('scripts/add_data_source.sh')
+  }
+  dependsOn: [
+    atlasUI
+    ohdsiWebApiWebapp
+    omopCDM
```

Comment:
it should depend on atlas security - as we need the admin user to be created.
The UI is not necessary here, `ohdsiWebApiWebapp` is included in atlas security and waiting for the CDM is not mandatory

Reply:
Fixed besides waiting for the cdm - it is mandatory since there is a new database being created there, if not succeeded no point in this data source

Reply:
BTW, left the ohdsiwebapi here - the deploymentAtlasSecurity is not really a dependency and I think logically it is more correct to keep it